content/en/news/releases/1.2.x/announcing-1.2/change-notes/index.md

```diff
@@ -30,7 +30,7 @@ aliases:
 - **Fixed** [SAN](https://tools.ietf.org/html/rfc5280#section-4.2.1.6) field in workload certificates is set as `critical`. This fixes the issue that some custom certificate verifiers cannot verify Istio certificates.
 - **Fixed** mutual TLS probe rewrite for HTTPS probes.
 - **Graduated** [SNI with multiple certificates support at ingress gateway](/docs/reference/config/networking/gateway/) from Alpha to Stable.
-- **Graduated** [certification management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds/) from Alpha to Beta.
+- **Graduated** [certification management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress/) from Alpha to Beta.
```

Comment:
I think you should point this to the archive as described here: https://preliminary.istio.io/about/contribute/remove-content/#reconcile-broken-links

Reply:
I am not against it but according to (my interpretation of) the flow chart this seems correct. There is a new document available (this one) so it says to link to the new doc

Reply:
That's step 2 of the flowchart. The first step is to check if the link is from an announcing-x.x doc and if YES, to point to the corresponding archive version

Reply:
Oops. thanks!